src/query/src/sql.rs

```diff
@@ -235,6 +249,100 @@ fn describe_column_semantic_types(
     ))
 }
 
+fn show_create_column_name(table_name: Vec<String>) -> VectorRef {
+    Arc::new(StringVector::from(table_name))
+}
+
+fn show_create_table_column_name(table_info: TableInfoRef) -> VectorRef {
+    let mut create_sql_column: Vec<String> = Vec::new();
+
+    let columns_schemas = table_info.meta.schema.column_schemas();
+    let primary_key_indices = &table_info.meta.primary_key_indices;
+    let table_name = &table_info.name;
+
+    create_sql_column.push(format!("CREATE TABLE {}(", table_name));
+    for (index, cs) in columns_schemas.iter().enumerate() {
+        create_sql_column.push(format!("{} ", cs.name.clone()));
+        create_sql_column.push(format!("{} ", cs.data_type.name()));
```

Comment:
The name of the data type is different from SQL's. For example, we use string type for SQL's char, varchar, and string types and float64 for SQL's double type. The conversion rule is in [statements.rs](https://github.com/GreptimeTeam/greptimedb/blob/develop/src/sql/src/statements.rs#L286).

It would be better if we convert the `ColumnSchema` into sqlparser's [`ColumnDef`](https://docs.rs/sqlparser/latest/sqlparser/ast/struct.ColumnDef.html), which implements `Display`, then format it.

You could find out how we parse `create table` sql in [create_parser.rs](https://github.com/GreptimeTeam/greptimedb/blob/c5b0d2431f4095c7a429e84124a594e4fd56df78/src/sql/src/parsers/create_parser.rs#L212)

Reply:
I find that only ColumnDef is supported in 'statements.rs.' to ColumnSchema. If I want to do the opposite, do I need to rewrite sqlparse? Then give 'CreateTable' impl 'Display' and 'fmt' out it.

Reply:
It should be possible to manually construct [`ColumnDef`](https://docs.rs/sqlparser/latest/sqlparser/ast/struct.ColumnDef.html) since all its fields are public.
```rust
fn column_schema_to_def(column_schema: &ColumnSchema) -> ColumnDef {
    ColumnDef {
        name: column_schema.name.clone(),
        data_type: concrete_data_type_to_sql_data_type(&column_schema.data_type), // Need to implement this func
        collation: None,
        options: vec![], // Here we also need to construct the options, based on `is_nullable` and `default_constraint`.
    }
}
```